.taskcluster.yml

```diff
@@ -1,127 +1,264 @@
+                                  then:
+                                      kind: cron-task
+                      routes:
+                          - checks
+                      scopes:
+                          # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
+                          $if: 'tasks_for == "github-push"'
+                          then:
+                              $let:
+                                  short_head_branch:
+                                      $if: 'head_branch[:11] == "refs/heads/"'
+                                      then: {$eval: 'head_branch[11:]'}
+                                      else: ${head_branch}
+                              in:
+                                  - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
+                          else:
+                              $if: 'tasks_for == "github-pull-request"'
+                              then:
+                                  - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
+                              else:
+                                  $if: 'tasks_for == "github-release"'
+                                  then:
+                                      - 'assume:repo:${repoUrl[8:]}:release'
+                                  else:
+                                      $if: 'tasks_for == "action"'
+                                      then:
+                                          # when all actions are hooks, we can calculate this directly rather than using a variable
+                                          - '${action.repo_scope}'
+                                      else:
+                                          - 'assume:repo:${repoUrl[8:]}:cron:${cron.job_name}'
+
+                      requires: all-completed
+                      priority: lowest
+                      retries: 5
+
+                      payload:
+                          env:
+                              # run-task uses these to check out the source; the inputs
+                              # to `mach taskgraph decision` are all on the command line.
+                              $merge:
+                                  - MOBILE_BASE_REPOSITORY: '${baseRepoUrl}'
+                                    MOBILE_HEAD_REPOSITORY: '${repoUrl}'
+                                    MOBILE_HEAD_REF: '${head_branch}'
+                                    MOBILE_HEAD_REV: '${head_sha}'
+                                    MOBILE_REPOSITORY_TYPE: git
+                                    TASKGRAPH_BASE_REPOSITORY: https://hg.mozilla.org/ci/taskgraph
+                                    TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
+                                    TASKGRAPH_HEAD_REV: ${taskgraph.revision}
+                                    TASKGRAPH_REPOSITORY_TYPE: hg
```

Comment:
We're not hg: is this correct?

Reply:
The taskgraph repository (which this is configuration for cloning) is still in hg. This also clones firefox-tv above (`MOBILE_*` prefixes), which has `git` as the repository type.

Reply:
:eyes: